### ISSUE TITLE
Ability to set show time on display

### ISSUE THREAD

**Issue by florin-ctu** [corvis/prana_rc#23]
Ability to set show time on display. 
Option to show time on display is for me and family a big help. The huge lighted display in each room is perfect for this, but unfortunately this can only be set from the IR Remote, at least I did not finde another way. 
This being said, the fact that this feature is missing from the phone app (at least iOS) will probably make this impossible to reverse engineer it.

**Comment by corvis** [comment]
Right, also my prana can display just speed and even doesn't have display to show time so there is no way for me to implement this. 

But I think we should keep this ticket open and maybe someday another developer who has prana with display and somespare time implement it. 

Also may I ask you to leave some information about your hardware here: https://github.com/corvis/prana_rc/discussions/17
